cmd/cli/config.go

```diff
@@ -8,6 +8,13 @@ type config struct {
 	LogFile  string          `yaml:"log_file"`
 	LogLevel string          `yaml:"log_level"`
 	Redis    cmd.RedisConfig `yaml:"redis"`
+	Cleanup  cleanupConfig   `yaml:"cleanup"`
+}
+
+type cleanupConfig struct {
+	Whitelist       []string `yaml:"whitelist"`
+	Delete          bool     `yaml:"delete"`
+	HandleAnonymous bool     `yaml:"handle_anonymous"`
```

Comment:
I don't understand this naming. What does "handle" mean in this case? Handle means delete? Or don't delete?

Reply:
Rename to CheckAnonymous